verigreen-collector-webapp/resources/config.properties

```diff
@@ -61,6 +61,19 @@ jenkinsparam.protected=false
 jenkinsparam.commitid=false
 jenkinsparam.committer=false
 
+#Enable offline FAQ page
```

Comment:
@borzamircea - please add a comment explaining what is the difference between offline and online mode.